baseapp_payments/rest_framework/payment_methods/serializers.py

```
@@ -0,0 +1,71 @@
+    pk = serializers.CharField(write_only=True, required=False)
+    default_payment_method_id = serializers.CharField(write_only=True, required=False)
+
+    def create(self, validated_data):
+        stripe_service = StripeService()
+        customer_id = validated_data.get("customer_id")
+        try:
+            setup_intent = stripe_service.create_setup_intent(
+                customer_id=customer_id,
+            )
+            return {"id": setup_intent["id"], "client_secret": setup_intent["client_secret"]}
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)